packages/block-editor/src/components/list-view/use-list-view-expand-selected-item.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../store';
+
+export default function useListViewExpandSelectedItem({
```

Comment:
Looks like the linting step is complaining that there's no space between the bracket and the brace!

```suggestion
export default function useListViewExpandSelectedItem( {
```